src/externalized/DefaultContentManager.cc

```diff
@@ -180,10 +178,8 @@ DefaultContentManager::DefaultContentManager(RustPointer<EngineOptions> engineOp
 	RustPointer<char> vanillaGameDir{EngineOptions_getVanillaGameDir(m_engineOptions.get())};
 
 	RustPointer<char> stracciatellaHome{EngineOptions_getStracciatellaHome()};
-	m_userHomeDir = stracciatellaHome.get();
 
-	RustPointer<char> dataDir{Fs_resolveExistingComponents(BASEDATADIR, vanillaGameDir.get(), true)};
-	m_dataDir = dataDir.get();
+	m_userPrivateFiles = std::unique_ptr<DirFs>(new DirFs(stracciatellaHome.get()));
```

Comment:
I think it's better and safer to use `std::make_unique<DirFs>(stracciatellaHome.get())`. As a bonus, it makes the line shorter.

Reply:
Fixed.